tests/task/traversal_data.py

```diff
@@ -69,7 +69,7 @@ def combined_mongo_posgresql_graph(
         connection_key=mongo_config.key,
     )
 
-    return (mongo_dataset, postgres_dataset)
+    return mongo_dataset, postgres_dataset
```

Comment:
weird, I thought tuples had to have enclosing `()`?

Reply:
apparently not, since the IDE pushed me to do it :)